### DISCUSSION TITLE
How about replacing the transmitter with AI?

### DISCUSSION THREAD

**Discussion by modestlyh** [NVlabs/sionna#616] [General]
I want to replace the transmitter with an neural network, but the channel is non-differentiable, so how should I train the network？I'm hoping for some help.

**Comment by SebastianCa** [comment]
Hi @modestlyh,
end-to-end learning without access to channel gradients is non-trivial task.  A possible solution is to use techniques from reinforcement learning, you can find an example in the [end-to-end learning notebook](https://nvlabs.github.io/sionna/examples/Autoencoder.html).

I am moving this thread to discussions as it is not a technical Sionna issue.
